src/components/form/form.stories.tsx

```diff
@@ -179,6 +179,9 @@ export const WithErrorsSummary: Story = () => (
   </Form>
 );
 WithErrorsSummary.storyName = "With Errors Summary";
+WithErrorsSummary.parameters = {
```

Comment:
Comment: it looks like this is creating issues with the storybook build and therefore chromatic isn't building

https://github.com/Sage/carbon/actions/runs/11030962735/job/30636704240